### ISSUE TITLE
Songs don't scrobble at all on versions >= 2.71

### ISSUE THREAD

**Issue by ArcanaOfSouls** [kawaiiDango/pScrobbler#262]
This might be an Android 13-specific bug, but I am impacted by a regression that started with version 2.71 and persisted in 2.72 (albeit with slightly different behavior). I only noticed it with 2.72 since I hadn't scrobbled anything since the 2.71 update. (I have auto-updates turned on in the Play Store.) In 2.72, I received no notification at all from Pano Scrobbler in Android's notification tray while any song was scrobbling, even though the appropriate permissions had been granted. Reinstalling the app did not resolve it. I went ahead and uninstalled 2.72 and manually installed the APK from GitHub for 2.71. In 2.71, the notification that a song was in the process of being scrobbled appeared, but nothing showed as scrobbling in Pano Scrobbler's interface. I checked Last.fm and nothing was being scrobbled there either. I uninstalled 2.71 and manually installed 2.70, and that version worked as expected (i.e. the notification appeared *and* the song was scrobbled).

For the moment, I'm stuck on version 2.70 since the newer versions don't scrobble anything at all.

If logs are useful, I can reinstall the newer versions at some point. But I can't guarantee a timeline since I'll have to uninstall 2.70, and newer versions just don't work at all for me.

**Comment by kawaiiDango** [comment]
Logs would be useful as I could not reproduce this on the Android 13 emulator

**Comment by ArcanaOfSouls** [comment]
I was attempting to grab logs, when I noticed that 2.72 worked properly this time. I've isolated the behavior (which also seems to impact 2.70) to the fact that we had an active media session in the Android 13 notification tray at the time of the app update. When we force quit all our music apps after the app upgraded, then force quit Pano Scrobbler and restarted our music streaming app, the app started working again, regardless of which of the last three versions we were using. Like I said, I have auto-updates turned on, so initially I wasn't even aware the app had been updated when this behavior occurred. I don't remember this occurring in prior Pano Scrobbler versions, but it may be an issue limited to Android 13.

Long story short: When Pano Scrobbler is newly installed and/or updated, if an app already has an active media session in the Android 13 notification tray (regardless of whether it's paused or playing), Pano Scrobbler has (in our case) failed to scrobble anything when navigating throughout the app in question. Force quitting both the media player and Pano Scrobbler resolves this.

If you can't reproduce after this update, let me know, and I can send logs. It's much less annoying now that I know what's going on though, especially since it's now easily worked around.

**Comment by ArcanaOfSouls** [comment]
Odd. I'm on a Pixel 6 Pro with the stock ROM for my device. Would logs be helpful at this point? If so, I can send them through the app. I am beginning to wonder if the issue has to do with the new media controls changing the format of the notification. Notable is that apps that haven't added native support for the new control format still utilize it but are missing features. My primary streaming service, Qobuz, has not been updated to support the feature yet, but YouTube Music has been. (Most music apps aren't updated to support the new feature yet since it requires an app to target API Level 33.) I'll test with YouTube Music and then repeat the test with Bandcamp (which hasn't been updated).

**Comment by TaprisSugarbell** [comment]
Xiaomi Redmi 9T
Android 10
MIUI 12.5.12
Version 2.72 and 2.70 they don't work on my device, at least it doesn't scroble anything.
The only way to do it is with the app's microphone.

**Comment by kawaiiDango** [comment]
Logs will be helpful, since I couldn't reproduce this on the Android 13
emulator.

**Comment by ArcanaOfSouls** [comment]
This hasn't been an issue for several updates now. An Android 13 security update may have fixed it in my case. Thanks for looking into it, and apologies for not circling back sooner. I'll close this issue.
